libvast/src/view.cpp

```diff
@@ -23,6 +23,7 @@ bool operator==(pattern_view x, pattern_view y) noexcept {
   return x.pattern_ == y.pattern_;
 }
 
+
```

Comment:
Extra newline.

Reply:
These are actually on purpose to delineate the different blocks of classes. Remove them?